lib/utils.pm

```diff
@@ -2826,4 +2827,21 @@ sub define_secret_variable {
     script_run("set +a");
 }
 
+=head2 ping_size_check
+    ping_size_check($target);
+ping_size_check will ping defined target with different and increasing size with disabled
+packet fragmentation.
+=cut
+
+sub ping_size_check {
+    my $target = shift;
+    # Check connectivity with different packet size to target
+    # Fragmentation is disabled, maximum size is 1430 to fit in 1458 MTU in GRE tunel
+    my @size = (100, 1000, 1350, 1400, 1430);
+    foreach (@size) {
+        assert_script_run("ping -M do -s $_ -c 1 $target");
+        record_info("PING OK", "Destination: $target Size: $_ ");
```

Comment:
that might be redundant with the `fail_message` that I suggested but if you like to have the test more verbose then this is ok of course